pkg/controller/podautoscaler/metrics/legacy_metrics_client_test.go

```diff
@@ -364,3 +404,7 @@ func testCollapseTimeSamples(t *testing.T) {
 	assert.InEpsilon(t, float64(75), val, 0.1, "collapsed sample value should be as expected")
 	assert.True(t, timestamp.Equal(now), "timestamp should be the current time (the newest)")
 }
+
+func calcTimestamp(t int) time.Time {
```

Comment:
this should have a way better name.  These tests are already hard to follow (maybe call it `offsetTimestampBy` or something).

Reply:
Fixed.